CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 
 ### Unreleased (BREAKING)
 
+## v2.0.0-beta4 (2020-10-05)
+Copy the debugbar fonts (font-awesome) on first render to htdocs.
```

Comment:
Could you format as a bullet list (`*`) entry? Some changelog parsers rely on that structure

Reply:
Fixed that up too, just lost in copy pasting from the commit